packages/coinstac-ui/app/render/components/dashboard-home.js

```diff
@@ -2,76 +2,32 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Alert } from 'react-bootstrap';
 import { connect } from 'react-redux';
-import { core, logger, notify } from 'ampersand-app';
-import { flatten, sortBy } from 'lodash';
+import { logger, notify } from 'ampersand-app';
 
 import StatusItem from './status-item.js';
 import { fetch as fetchProjects } from '../state/ducks/projects.js';
 import { fetchComputations } from '../state/ducks/computations.js';
+import { fetch as fetchConsortiaResults } from '../state/ducks/remote-results';
 
 class DashboardHome extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      remoteResults: [],
-    };
-  }
-
   componentWillMount() {
-    const { dispatch } = this.props;
+    const { computations, dispatch, remoteResults } = this.props;
+
+    if (!remoteResults.length) {
+      dispatch(fetchConsortiaResults());
+    }
+
+    if (!computations.length) {
+      dispatch(fetchComputations());
+    }
 
-    dispatch(fetchComputations());
     // TODO: Modify action creator to use Promise
     dispatch(fetchProjects((error) => {
       if (error) {
         logger.error(error);
         notify('error', error.message);
       }
     }));
-
-    this.getRemoteResults().then(() => {
-      this.interval = setInterval(() => this.getRemoteResults(), 2000);
-    });
-  }
-
```

Comment:
Much of this section of code deals with setting and clearing out the *remoteResults* interval. Since *remoteResults* are now being pulled from redux, the timer could just call that action. I just wanted to double-check that there isn't some back-end process that could update this before I put the interval timer back in.